examples/src/bin/composite_template.rs

```diff
@@ -0,0 +1,114 @@
+//! # Composite Template Example
+//!
+//! This sample demonstrates how to create a widget using GTK's composite templates.
+
+#[macro_use]
+extern crate glib;
+#[macro_use]
+extern crate gtk;
```

Comment:
```suggestion
```

Reply:
Indeed, I missed the Rust 2015-isms in the example. Thanks :)